tests/unit/services/ember-cli-test.js

```diff
@@ -65,6 +65,21 @@ test("buildProperties() works as expected with replacements", function (assert)
   assert.ok(props.content.indexOf('helper(myHelper)') !== -1, 'Replacements worked if multiple');
 });
 
+test("buildProperties('model') creates a valid file for a model", function(assert) {
```

Comment:
Yes. If you'd like, you can instead write an acceptance test checking that a model can be added, in gist-test.js